packages/slo-stat-bot/utils/schema.json

```diff
@@ -0,0 +1,107 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "SLO rules",
```

Comment:
I think right now we don't copy over the whole package directory during the cloud build step, we should either update [this script](https://github.com/googleapis/repo-automation-bots/blob/master/scripts/build-functions.sh#L37) so that we copy over everything, including `utils`.